GameReviewBackend/API/Program.cs

```diff
@@ -167,7 +164,15 @@ public static void Main(string[] args)
             app.UseAuthentication();
             app.UseAuthorization();
 
-            app.UseMiddleware<GlobalExceptionHandlingMiddleware>();
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
```

Comment:
minor but I think I would switch this logic around so if its == "production" then use prod

Reply:
Switched these around